src/components/FooterPreLogin/FooterPreLogin.tsx

```diff
@@ -98,35 +110,34 @@ export const FooterPreLogin = ({
           </Grid>
           <Grid item xs={12} sm={3}>
```

Comment:
this entire component should be hidden if the url has not been provided, in order to not render anything if there is not a URL where to fetch